pkg/toolsets/kubevirt/vm/stop/tool.go

```diff
@@ -0,0 +1,123 @@
+package stop
+
+import (
+	"fmt"
+
+	"github.com/containers/kubernetes-mcp-server/pkg/api"
+	"github.com/containers/kubernetes-mcp-server/pkg/output"
+	"github.com/google/jsonschema-go/jsonschema"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/utils/ptr"
+)
+
+func Tools() []api.ServerTool {
+	return []api.ServerTool{
+		{
+			Tool: api.Tool{
```

Comment:
What do you think about making a new common function in utils, that will return `api.Tool` from `getCommonTool(name, desc)`?

To have this big `api.Tool` definition shorter here, simply by:

```
Tool: utils.getCommonTool(name, desc)
```